src/ipcache.cc

```diff
@@ -320,7 +320,7 @@ ipcacheRelease(ipcache_entry * i, bool dofree)
 static ipcache_entry *
 ipcache_get(const char *name)
 {
-    if (ip_table != nullptr)
+    if (ip_table)
```

Comment:
Please revert this out-of-scope change since the PR no longer modifies this code for in-scope reasons:

```suggestion
    if (ip_table != nullptr)
```